app/controllers/state_file/questions/landing_page_controller.rb

```diff
@@ -6,7 +6,6 @@ class LandingPageController < QuestionsController
       skip_before_action :redirect_if_in_progress_intakes_ended
 
       def edit
-        @state_name = StateFileBaseIntake::STATE_CODE_AND_NAMES[params[:us_state]]
```

Comment:
I think this is actually used in landing_page/edit.html.erb ?

Reply:
oh good catch! forgot to change the view to just call the helper `state_name` instead of the instance variable